### ISSUE TITLE
Styling issue with the Top Results table. 

### ISSUE THREAD

**Issue by Javano** [TheRealDF/ADFGX_Cloud_Solver#11]
The results table styling is all out of wack. It shouldn't be all bolded, and the columns aren't in the proper order. 

I'll submit a fix once #9 is all merged in.

**Comment by TheRealDF** [comment]
When you submitted the pull, there was no "result" column in it, so i had to add it. Though i really dont think order is that important. Nor bold letters.

**Comment by Javano** [comment]
Yeah, the DGV config seems to be oddly stubborn for me too. It seemed to reset the settings every time I changed any of the underlying datasource mapping; I assume it was inheriting some parent style you had set elsewhere.

Of course it's not that important, hence the "I'll fix it later" part. _(I'm just a perfectionistic PoS and I had it styled nicely so that it all fit perfectly on the form, without needing a horizontal scrollbar. :P)_

**Comment by TheRealDF** [comment]
haha i get it, sadly i was in a rush, and didnt want to do the math involved with putting a column with the result in myself. The result has to be fully open and readable because its the most important out of all of them
